be/src/olap/rowset/segment_v2/column_zone_map.cpp

```diff
@@ -82,6 +82,8 @@ void ColumnZoneMapBuilder::_reset_zone_map() {
     Slice *min_slice = (Slice *)_zone_map.min_value;
     min_slice->data = _max_string_value;
     min_slice->size = OLAP_STRING_MAX_LENGTH;
+    Slice *max_slice = (Slice *)_zone_map.max_value;
+    max_slice->size = 0;
```

Comment:
do not do this, because the column maybe not char/varchar type.
and set_to_min(_zone_map.max_value) will set the initial value of max_value to min.

Reply:
max_value/min_value 's memory comes char array of arena,but the content of char array is unknown,it's necessary to init the array first,or the size of slice will be very large, then core dump happends

Reply:
1. the char array allocated to min_value and max_value must be initialed(like set to 0),otherwise when casting array to slice ,the size property of slice may be very huge
2. I think do the init in column_zone_map is better,because column_zone_map contains the context(like _max_string_value,arena),only do init when type=char